cli/flox/src/commands/init/mod.rs

```diff
@@ -809,4 +817,63 @@ mod tests {
             activate_mode: None,
         });
     }
+
+    /// Verify that format_customization() correctly converts InitCustomization to TOML.
+    #[test]
+    fn test_format_customization() -> Result<()> {
+        // Create a test InitCustomization with various fields populated
+        let customization = InitCustomization {
+            hook_on_activate: Some("echo 'Activating environment'".to_string()),
+            profile_common: Some("export COMMON_VAR=value".to_string()),
+            profile_bash: Some("export BASH_VAR=value".to_string()),
+            profile_fish: Some("set -x FISH_VAR value".to_string()),
+            profile_tcsh: Some("setenv TCSH_VAR value".to_string()),
+            profile_zsh: Some("export ZSH_VAR=value".to_string()),
+            packages: Some(vec![CatalogPackage {
+                id: "test-package".to_string(),
+                pkg_path: "test.package".to_string(),
+                version: Some("1.0.0".to_string()),
+                systems: None,
+            }]),
+            activate_mode: None,
+        };
+
+        let toml_str = format_customization(&customization)?;
+        
```

Comment:
There's some additional whitespace that `rustfmt` is upset about here: https://github.com/flox/flox/actions/runs/14920997810/job/41916206169#step:9:481
```suggestion
```

It's worth configuring your editor to format automatically on save so that you don't have to think about it. It also often gives an early indication when code doesn't do what you originally intended, such as misplaced braces and method chains.

I've raised https://github.com/flox/flox/pull/3057 to improve feedback from the failure.